### PR TITLE
Add ShortList sniff

### PR DESCRIPTION
**Pull request by carusogabriel**
Small one: `list()` to `[]`.

Ref: https://github.com/slevomat/coding-standard/pull/300

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
         "php": "^7.1",
         "squizlabs/php_codesniffer": "^3.2.3",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.4.2",
-        "slevomat/coding-standard": "^4.5.0"
+        "slevomat/coding-standard": "dev-master"
```
Comment (Ocramius) [inline composer.json]: Obviously to be changed before merging

**Comment by carusogabriel** [comment]
Rebasead and updated to Slevomat CS 4.6 :+1:

File: tests/expected_report.txt [hunk 2]
```diff
@@ -8,7 +8,7 @@ tests/input/concatenation_spacing.php                 24      0
 tests/input/EarlyReturn.php                           4       0
 tests/input/example-class.php                         22      0
 tests/input/forbidden-comments.php                    4       0
-tests/input/forbidden-functions.php                   6       0
+tests/input/forbidden-functions.php                   7       0
```
Comment (Majkl578) [inline tests/expected_report.txt]: This change looks unrelated, where does it come from?

Comment (Majkl578) [reply tests/expected_report.txt]: Maybe doesn't require separate file, but it doesn't make sense to have it in "forbidden-functions.php". "example-class.php" would be better place if you really want to avoid new file.

**Comment by Ocramius** [comment]
:ship:
